x/rewards/module.go

```diff
@@ -30,6 +30,9 @@ var (
 	_ module.AppModule      = AppModule{}
 )
 
+// ConsensusVersion defines the current x/rewards module consensus version.
+const ConsensusVersion = 1
```

Comment:
Is consensus version 1 safe? or should it be 2? Not sure where the SDK starts from on cons versions tbh.

Reply:
Oops thanks for the catch